h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -108,8 +108,8 @@ public boolean isSupervised() {
     };
   }
 
-  @Override public boolean havePojo() { return true; }
-  @Override public boolean haveMojo() { return true; }
+  @Override public boolean havePojo() { return !_parms._HGLM; }
+  @Override public boolean haveMojo() { return !_parms._HGLM; }
```

Comment:
@wendycwong can you please confirm that this is ok?

`estimator_base` used to have the following logic that I really wanted to remove:
```python
        if (model_json["algo"]=="glm") and self.HGLM:
            m._have_pojo = False
            m._have_mojo = False
        else:
            m._have_pojo = model_json.get('have_pojo', True)
            m._have_mojo = model_json.get('have_mojo', True)
```
so I figured it would be better that have the values changed on server side.

Reply:
This looks good to me.  Thanks.

Reply:
Seb:  Can you please move this to GLMModel.java?  Take a look at the haveMojo/Pojo there and proceed accordingly.  Thanks, Wendy